src/server_manager/web_app/digitalocean_server.ts

```diff
@@ -385,32 +381,26 @@ export class DigitaloceanServerRepository implements server.ManagedServerReposit
     const onceKeyPair = crypto.generateKeyPair();
     const watchtowerRefreshSeconds = this.image ? 30 : undefined;
     const installCommand = getInstallScript(
-        this.digitalOcean.accessToken, name, this.image, watchtowerRefreshSeconds, this.metricsUrl, this.sentryApiUrl);
+        this.digitalOcean.accessToken, name, this.image, watchtowerRefreshSeconds, this.metricsUrl,
+        this.sentryApiUrl);
 
     const dropletSpec = {
       installCommand,
       size: MACHINE_SIZE,
       image: 'docker',
       tags: [SHADOWBOX_TAG],
     };
-    return onceKeyPair.then((keyPair) => {
-      if (this.debugMode) {
-        // Strip carriage returns. They produce annoying blank lines when pasting
-        // into a terminal.
-        console.log(
```

Comment:
This is quite useful. Why are you killing int?

Reply:
Funny, I fought to keep this bit of code during one of our security audits and I think we still reasonable safely can as it only triggers in debug mode but recently I've found it much easier, when debugging stuff on droplets, to hack `makeCreateDropletRequest` to not send a key. I don't feel very strongly.

Reply:
I use it often. It's easier than editing the code or resetting the password. I'd prefer to keep it

Reply:
Sure. Even though this is an `if`/`else` block I felt better doing this after also disabling breadcrumbs for `console.debug` statements.